code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -527,3 +527,23 @@
 			dat += "[new_text]\n" //dat.Join("\n") doesn't work here, for some reason
 	if(dat.len)
 		return dat.Join()
+
+/mob/living/carbon/human/examine_more(mob/user)
+	. = ..()
+	if ((wear_mask && (wear_mask.flags_inv & HIDEFACE)) || (head && (head.flags_inv & HIDEFACE)))
+		return
+	var/age_text
+	switch(age)
+		if(-INFINITY to 25)
+			age_text = "very young"
+		if(26 to 35)
+			age_text = "of adult age"
```

Comment:
```suggestion
		if(-INFINITY to 17) // SKYRAT EDIT ADD START -- AGE EXAMINE
			age_text = "a child"
		if(18 to 25)
			age_text = "a young adult" // SKYRAT EDIT END
		if(26 to 35)
			age_text = "of adult age"
```

Reply:
I don't like actual children and adults up to the age of 25 being lumped in together, especially under the label "very young"

Reply:
No!

No one should be able to set their age below 18, due to mature content. We most definitely do not want a mention of child.

Reply:
sure, updated recommendation

Reply:
That first bracket serves no purpose.